setup.py

```diff
@@ -45,7 +45,7 @@ def find_version(*file_paths):
     'ansimarkup>=1.0.0',
     'colorama==0.4.*',
     'graphene==2.1.6',
-    'isodatetime==1!2.0.*',
+    'metomi-isodatetime==1!2.0.*',
```

Comment:
As it is not published yet, this would fail in my local environment (not sure about Travis). Which means that we are postponing the merge until this is released?

Reply:
Ay! Yes!